howso/visuals/visuals.py

```diff
@@ -581,3 +581,48 @@ def plot_interpretable_prediction(
 
     if len(figures) == 1:
         return figures[0]
+
+
+def plot_fairness_disparity(
+    fairness_results: dict,
+    ref: str,
```

Comment:
I think you can afford to be verbose here: `reference_class` isn't that much longer and I don't have to scroll down to know what it might mean.